### PR TITLE
endpoint for Master ID project

### PR DESCRIPTION
**Pull request by Vlad-CL**
# JIRA

https://clxmedia.atlassian.net/browse/CLXD-1689

# Description

API endpoint for Master ID data. 
Odoo will return information regarding the connections configured for client(c) and budgets for connected subscriptions

### CODE REVIEW

File: clx_subscription_creation/controllers/portal.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def client_platform_spend(
             return {"status": 404, "response": {"error": "Access Token is wrong"}}
 
         if not partners_id:
-            return {"status": 404, "response": {"error": "partner_id is mandratory parameter."}}
+            return {"status": 404, "response": {"error": "partner_id is mandatory parameter."}}
```
Comment (toddcd) [inline clx_subscription_creation/controllers/portal.py:181]: should this be a 400 Bad Request?

File: clx_subscription_creation/controllers/portal.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def client_platform_spend(
             return {"status": 404, "response": {"error": "Access Token is wrong"}}
```
Comment (toddcd) [inline clx_subscription_creation/controllers/portal.py:178]: this should be 401 or 403

**Review by toddcd — APPROVED** [review]

File: clx_subscription_creation/controllers/portal.py (hunk 2 above)
Comment (dmontooth) [reply clx_subscription_creation/controllers/portal.py:178]: This should be 401. 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401

File: clx_subscription_creation/controllers/portal.py (hunk 1 above)
Comment (dmontooth) [reply clx_subscription_creation/controllers/portal.py:181]: Yes. This should be 400.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400

**Review by dmontooth — CHANGES_REQUESTED** [review]
Need to update a couple of HTTP response codes

File: clx_subscription_creation/controllers/portal.py (hunk 2 above)
Comment (Vlad-CL) [reply clx_subscription_creation/controllers/portal.py:178]: Good point, thank you

File: clx_subscription_creation/controllers/portal.py (hunk 1 above)
Comment (Vlad-CL) [reply clx_subscription_creation/controllers/portal.py:181]: True

**Comment by Vlad-CL** [comment]
response codes updated

**Review by dmontooth — APPROVED** [review]

**Pull request merged**